### PR TITLE
fix(curriculum): Clarify definition of "symmetric difference"

### PR DESCRIPTION
**Pull request by TyMick**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

When completing [Algorithms: Find the Symmetric Difference](https://www.freecodecamp.org/learn/coding-interview-prep/algorithms/find-the-symmetric-difference), after reading the original description, I didn't realize that symmetric difference was a binary operation, so I had to go [to Wikipedia](https://en.wikipedia.org/wiki/Symmetric_difference) to find out why, when three arrays are provided, an element present in all three of them should be part of their collective symmetric difference.

Screen Shot 2020-05-11 at 5 06 08 PM

Not that requiring extra research is a bad thing, but I feel like a clearer provided definition would be better.

### Original description:

> Create a function that takes two or more arrays and returns an array of the <dfn>symmetric difference</dfn> (<code>&xutri;</code> or <code>&oplus;</code>) of the provided arrays.
> 
> Given two sets (for example set <code>A = {1, 2, 3}</code> and set <code>B = {2, 3, 4}</code>), the mathematical term "symmetric difference" of two sets is the set of elements which are in either of the two sets, but not in both (<code>A &xutri; B = C = {1, 4}</code>). For every additional symmetric difference you take (say on a set <code>D = {2, 3}</code>), you should get the set with elements which are in either of the two the sets but not both (<code>C &xutri; D = {1, 4} &xutri; {2, 3} = {1, 2, 3, 4}</code>). The resulting array must contain only unique values (<em>no duplicates</em>).

### New description:

> Create a function that takes two or more arrays and returns an array of the <dfn>symmetric difference</dfn> (<code>&xutri;</code> or <code>&oplus;</code>) of the provided arrays.
> 
> The mathematical term "symmetric difference" of two sets (for example, set <code>A = {1, 2, 3}</code> and set <code>B = {2, 3, 4}</code>) is the set of elements which are in either of the two sets, but not in both (<code>A &xutri; B = {1, 4}</code>).
> 
> Symmetric difference is a binary operation, so to evaluate an expression involving symmetric differences between three elements (<code>A &xutri; B &xutri; C</code>), you must complete one operation at a time (thus, for <code>A</code> and <code>B</code> above, and <code>C = {2, 3}</code>, <code>A &xutri; B &xutri; C = (A &xutri; B) &xutri; C = {1, 4} &xutri; {2, 3} = {1, 2, 3, 4}</code>).
> 
> The resulting array must contain only unique values (<em>no duplicates</em>).

### CODE REVIEW

File: curriculum/challenges/english/08-coding-interview-prep/algorithms/find-the-symmetric-difference.english.md [hunk 1]
```diff
@@ -8,7 +8,9 @@ forumTopicId: 301611
...
+The mathematical term "symmetric difference" of two sets (for example, set <code>A = {1, 2, 3}</code> and set <code>B = {2, 3, 4}</code>) is the set of elements which are in either of the two sets, but not in both (<code>A &xutri; B = {1, 4}</code>).
+Symmetric difference is a binary operation, so to evaluate an expression involving symmetric differences between three elements (<code>A &xutri; B &xutri; C</code>), you must complete one operation at a time (thus, for <code>A</code> and <code>B</code> above, and <code>C = {2, 3}</code>, <code>A &xutri; B &xutri; C = (A &xutri; B) &xutri; C = {1, 4} &xutri; {2, 3} = {1, 2, 3, 4}</code>).
```
Comment (erictleung) [inline curriculum/challenges/english/08-coding-interview-prep/algorithms/find-the-symmetric-difference.english.md:12]: Clarify what is a binary operation.

```suggestion
Symmetric difference is a binary operation, which means it operates only on two things at a time. So to evaluate an expression involving symmetric differences between three elements (<code>A &xutri; B &xutri; C</code>), you must complete one operation at a time (thus, for <code>A</code> and <code>B</code> above, and <code>C = {2, 3}</code>, <code>A &xutri; B &xutri; C = (A &xutri; B) &xutri; C = {1, 4} &xutri; {2, 3} = {1, 2, 3, 4}</code>).
```

**Review by erictleung — CHANGES_REQUESTED** [review]
Really loving the changes. Just a quick suggestion. Let me know what you think.

**Comment by tywmick** [comment]
@erictleung I like it! And then for one more slight wording change (I didn't realize my committing your suggestion would close that suggestion conversation), lemme make one more quick commit...

**Comment by tywmick** [comment]
Taking out "at a time" from the first sentence since the phrase is in the second sentence, swapping "things" in favor of "elements", and emphasizing the word "three" in the second sentence after talking about binary/two in the previous sentence.

File: curriculum/challenges/english/08-coding-interview-prep/algorithms/find-the-symmetric-difference.english.md [hunk 2]
```diff
@@ -8,7 +8,9 @@ forumTopicId: 301611
...
+The mathematical term "symmetric difference" of two sets (for example, set <code>A = {1, 2, 3}</code> and set <code>B = {2, 3, 4}</code>) is the set of elements which are in either of the two sets, but not in both (<code>A &xutri; B = {1, 4}</code>).
+Symmetric difference is a binary operation, which means it operates on only two elements. So to evaluate an expression involving symmetric differences between <em>three</em> elements (<code>A &xutri; B &xutri; C</code>), you must complete one operation at a time (thus, for <code>A</code> and <code>B</code> above, and <code>C = {2, 3}</code>, <code>A &xutri; B &xutri; C = (A &xutri; B) &xutri; C = {1, 4} &xutri; {2, 3} = {1, 2, 3, 4}</code>).
```
Comment (erictleung) [inline curriculum/challenges/english/08-coding-interview-prep/algorithms/find-the-symmetric-difference.english.md:12]:
```suggestion
Symmetric difference is a binary operation, which means it operates on only two elements. So to evaluate an expression involving symmetric differences among <em>three</em> elements (<code>A &xutri; B &xutri; C</code>), you must complete one operation at a time (thus, for <code>A</code> and <code>B</code> above, and <code>C = {2, 3}</code>, <code>A &xutri; B &xutri; C = (A &xutri; B) &xutri; C = {1, 4} &xutri; {2, 3} = {1, 2, 3, 4}</code>).
```

Comment (erictleung) [reply curriculum/challenges/english/08-coding-interview-prep/algorithms/find-the-symmetric-difference.english.md:12]: @tywmick between is for pairs, among is for more. Thoughts?

**Comment by erictleung** [comment]
> 
> 
> Taking out "at a time" from the first sentence since the phrase is in the second sentence, swapping "things" in favor of "elements", and emphasizing the word "three" in the second sentence after talking about binary/two in the previous sentence.

Ah nice catches. Yeah, "things" didn't sound right and I couldn't come up with a better word 😅 

I read it again and found another thing. I commented above. Let me know what you think.

File: curriculum/challenges/english/08-coding-interview-prep/algorithms/find-the-symmetric-difference.english.md (hunk 2 above)
Comment (tywmick) [reply curriculum/challenges/english/08-coding-interview-prep/algorithms/find-the-symmetric-difference.english.md:12]: Ah good catch!

**Review by erictleung — APPROVED** [review]
Great! LGTM! 👍 Thanks for working on this. If I remember correctly, I initially put those symbols in and tried my best to word this challenge. Thanks for taking the time and thought to improve it. I think it is much better ✨

**Comment by tywmick** [comment]
My pleasure! 😊

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
As Eric said, this is really nice.  The only thing that I could think of was that it might be better to put lines 10 and 13 ('Create a function...' and 'The resulting array...') in the instructions section.  This will necessitate a little rewording, since the `<dfn>` and the symbols still should be introduced first.

Does that sound reasonable?

**Comment by tywmick** [comment]
@ojeytonwilliams Sounds good! Here are the new sections:

> ## Description
> 
> The mathematical term <dfn>symmetric difference</dfn> (<code>&xutri;</code> or <code>&oplus;</code>) of two sets is the set of elements which are in either of the two sets but not in both. For example, for sets <code>A = {1, 2, 3}</code> and <code>B = {2, 3, 4}</code>, <code>A &xutri; B = {1, 4}</code>.
> 
> Symmetric difference is a binary operation, which means it operates on only two elements. So to evaluate an expression involving symmetric differences among <em>three</em> elements (<code>A &xutri; B &xutri; C</code>), you must complete one operation at a time. Thus, for sets <code>A</code> and <code>B</code> above, and <code>C = {2, 3}</code>, <code>A &xutri; B &xutri; C = (A &xutri; B) &xutri; C = {1, 4} &xutri; {2, 3} = {1, 2, 3, 4}</code>.
> 
> ## Instructions
> 
> Create a function that takes two or more arrays and returns an array of their symmetric difference. The returned array must contain only unique values (<em>no duplicates</em>).

**Review by ojeytonwilliams — APPROVED** [review]

**Comment by ojeytonwilliams** [comment]
The challenge looks great now, thanks for taking the time to make these improvements!

**Pull request merged**